textpipe/doc.py

```diff
@@ -514,14 +514,14 @@ def aggregate_word_vectors(self, model_name=None, aggregation='mean', normalize=
 
     def _load_gensim_word2vec_model(self, model_file=None):
         """
-        Loads pre-trained Gensim word2vec model
-        >>> model = Doc('')._load_gensim_word2vec_model('tests/models/gensim_test_nl.w2v')
+        Loads pre-trained Gensim word2vec keyed vector model
+        >>> model = Doc('')._load_gensim_word2vec_model('tests/models/gensim_test_nl.kv')
         >>> type(model)
         <class 'gensim.models.keyedvectors.Word2VecKeyedVectors'>
         """
         lang = self.language if self.is_reliable_language else self.hint_language
         if not self._gensim_vectors or lang not in self._gensim_vectors:
-            vectors = Word2Vec.load(model_file).wv
+            vectors = KeyedVectors.load(model_file, mmap='r')
```

Comment:
Is there any possible issue with this `mmap='r'` thingy? 

> If the file being loaded is compressed (either ‘.gz’ or ‘.bz2’), then `mmap=None must be set. 
> https://radimrehurek.com/gensim/models/keyedvectors.html

We could add a check for this?

Reply:
As far as I know, compressing these files is not common, and I think we can rely on Gensim throwing an error for this.